src/containers/HomePage.spec.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { fromJS } from 'immutable';
+// import nock from 'nock';
+
+import HomePage from './HomePage';
+import configureStore from '../store/configureStore';
+import { withStore } from '../utils/testHelpers';
+
+describe('<HomePage />', () => {
```

Comment:
This spec is a little vague. What do we expect to see when arriving to the homepage? One left menu component and a map component, right? Can you test on the presence of those?